packages/truffle-db/src/workspace/test/index.ts

```diff
@@ -57,66 +109,64 @@ mutation AddSource($contents: String!, $sourcePath: String) {
   }
 }`;
 
-it("adds source", async () => {
-  const workspace = new Workspace();
-  const variables = {
-    contents: Migrations.source,
-    sourcePath: Migrations.sourcePath,
-    id: soliditySha3(Migrations.source, Migrations.sourcePath)
-  }
-
-  // add source
-  {
-    const result = await graphql(
-      schema, AddSource, null, { workspace }, variables
-    );
+describe("Source", () => {
+  it("adds source", async () => {
+    const client = new WorkspaceClient();
 
-    const { data } = result;
-    expect(data).toHaveProperty("sourcesAdd");
+    const variables = {
+      contents: Migrations.source,
+      sourcePath: Migrations.sourcePath,
+      id: soliditySha3(Migrations.source, Migrations.sourcePath)
```

Comment:
Let's take this out of `variables` and make it a separate variable, something like:

```javascript
const expectedId = generateId({
  source: Migrations.source,
  sourcePath: Migrations.sourcePath
});
```